r2-core/src/test/java/test/r2/transport/http/client/TestAsyncPool.java

```diff
@@ -534,7 +544,7 @@ public void testGetStatsWithErrors() throws Exception
     Assert.assertEquals(stats.getCheckedOut(), GET - PUT_BAD - DISPOSE);
     // When the each create fails, it will retry and cancel the waiter,
     // resulting in a second create error.
```

Comment:
This comment is not updated anymore

Reply:
Good Catch. removed non relevant comments. Thanks